internal/pkg/agent/application/coordinator/coordinator.go

```diff
@@ -1490,10 +1496,11 @@ func (c *Coordinator) processConfigAgent(ctx context.Context, cfg *config.Config
 		return err
 	}
 
-	// Disabled for 8.8.0 release in order to limit the surface
-	// https://github.com/elastic/security-team/issues/6501
-
-	// c.setProtection(protectionConfig)
+	protectionConfig, err := protection.GetAgentProtectionConfig(m)
+	if err != nil && !errors.Is(err, protection.ErrNotFound) {
+		return fmt.Errorf("could not read the agent protection configuration: %w", err)
+	}
+	c.setProtection(protectionConfig)
```

Comment:
Since you are reintroducing this code (by uncommenting it), should the following section also be uncommented/reintroduced? https://github.com/elastic/elastic-agent/blob/64e5457f653c1f11a7fc60d07b5adeaf6092a2f3/internal/pkg/agent/application/actions/handlers/handler_action_application.go#L53-L66

Reply:
no this is different action that we should not touch at this step

Reply:
Yeah, it's a different action but does it need to also start doing the signature validation?  Just clarifying if this is something we should a) do in a follow up PR or b) not touch at all?

Reply:
we should do this in a followup
linked issue says:
> The infrastructure for action signing was implemented as part of the tamper protection implementation and should be enabled as part of this work, initially scoped only to the migrate action to reduce risk.